src/main/resources/org/jenkinsci/plugins/configfiles/folder/FolderConfigFileAction/configfiles.jelly

```diff
@@ -20,16 +20,20 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+${it.CONFIGURE}
 -->
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
 		 xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:cf="/lib/configfiles">
-	<l:layout permission="${app.ADMINISTER}" norefresh="true">
+	<l:layout permission="${it.CONFIGURE}" norefresh="true">
+
 
 		<cf:sitepanel />
 
 		<l:main-panel>
+    		<j:out value="${it.permissionGranted}" />
```

Comment:
i was trying to figure out how to prevent displaying the links entirely instead of showing the access exceptions. e.g. when a user has no permission to configure config files on a folder, IMHO, the link should not be displayed.